### ISSUE TITLE
Images, which I can't remove

### ISSUE THREAD

**Issue by azproduction** [BookMePhoto/bookme#2]
![screen shot 2018-02-28 at 22 28 32](https://user-images.githubusercontent.com/483865/36814065-c6a801b2-1cd6-11e8-9cb4-97adabdcd2a7.png)

Image ID:
```
portfolio_remove('f47dd730428ec213a4892a248d0d03f4.jpg');
```

Gives Me 404 https://bookme.photo/images/profile/portfolio/f47dd730428ec213a4892a248d0d03f4.jpg

Seems there were some issues during batch upload.

Also when form is submitted the correct image is marked to be deleted
```
------WebKitFormBoundaryyZvTFF0rY2pbdq8w
Content-Disposition: form-data; name="portfolio_img_delete"

f47dd730428ec213a4892a248d0d03f4.jpg
```

**Comment by azproduction** [comment]
There is similar case on FB, to which was replied "You had internet problem during uploading photos." Which is completely false statement.

**Comment by azproduction** [comment]
I suppose there were many cases like that and only few were reported. So it would be great if we generally cleanup similar buggy state images.

**Comment by spajo** [comment]
@azproduction Hey, I cannot reproduce it, I tried 
- uploading 10 at once
- removing X and adding X at once
- adding X then adding Y 
several times.

**Comment by azproduction** [comment]
Maybe there was some race condition under heavy load when many pictures were uploaded in the same time, dunno :)

**Comment by spajo** [comment]
@azproduction Our dev said that he implemented a fix on a testing environment, but he doesn't really know how to test it (reproduce the race/network problem), but it should work he says.

Also, what do you mean clean up? Clean up broken photos?

**Comment by azproduction** [comment]
> Clean up broken photos?
@spajo yes, because they are taking "one photo"-space without any value

**Comment by spajo** [comment]
@azproduction Our dev told me that it isn't quite possible because it would mean checking every image if it isn't corrupted. I'll try writing some script to test the photos and if that proves successful we will have all the corrupted photos removed. He said that people are actively reporting broken photos and he is dealing with it. He also said that the issue should not repeat as of now.

**Comment by azproduction** [comment]
@spajo image is actually not corrupted, but not written to the disk. https://bookme.photo/images/profile/portfolio/f47dd730428ec213a4892a248d0d03f4.jpg – 404

**Comment by spajo** [comment]
@azproduction sorry, I didn't mention that broken images are either corrupted or not uploaded.

**Comment by azproduction** [comment]
@spajo  There still at least some images not fixed. :( On my account "azproduction" there are 2 blank ones. Could you help with this?

**Comment by spajo** [comment]
@azproduction sure, I'll see what I can do

**Comment by azproduction** [comment]
@spajo that would be great. We almost closed registration, but still there are so many people can't upload 10 pictures.

**Comment by azproduction** [comment]
@spajo somethings changed recently. I had 2 images which I can't delete now I have 3 :(

**Comment by spajo** [comment]
@azproduction Okay, forwarding this now.

**Comment by spajo** [comment]
@azproduction They said to me that it's done.
